### PR TITLE
[media / Test Plan] Update TestPlan.md

### LINKED ISSUE #6336: [media] Update test plan to remove mention of "site" field
**Describe the bug**
The site selection has been removed in https://github.com/aces/Loris/pull/5577 but the test plan was never updated.

### PR DESCRIPTION
**Pull request by racostas**
Media test plan update

* Resolves #6336

### CODE REVIEW

File: modules/media/test/TestPlan.md [hunk 1]
```diff
@@ -51,15 +51,13 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
     - A popup should prompt you to select a **Visit Label** as it is a required field ❌
   3. Select Visit Label and click on the 👉  **Upload file** again
     - A popup should prompt you to select a **Site** as it is a required field ❌
-  4. Select Site and click on the 👉  **Upload file** again
-    - A popup should prompt you to select a **File to upload** as it is a required field ❌
-  5. Click on the 👉 **Browse** button and a select a file from your file system
+  4. Click on the 👉 **Browse** button and a select a file from your file system
     - Note: it is suggested to try different file types and sizes (e.g pdf, mp4, mov, jpg, doc, etc)
-  6. After you selected the file click on **Upload File** one more time
+  5. After you selected the file click on **Upload File** one more time
```
Comment (maltheism) [inline modules/media/test/TestPlan.md:56]: The "one more time" should be removed and I think the behaviour of "popup prompting to name the file" was removed.

Comment (racostas) [reply modules/media/test/TestPlan.md:56]: @maltheism I think the "popup prompting to name the file" refers to the case the file didn't  have been named properly before.

image

Comment (racostas) [reply modules/media/test/TestPlan.md:56]: regarding to the "one more time" statement I think is a question of style from the original document. I'm don't have any feeling against, neither in favour of removed it or leave it. In the case we take the first option, should we remove also the "again" statements in the following lines?   

2. Select PSCID and click on the 👉 Upload file button `again` - A popup should prompt you to select a Visit Label as it is a required field ❌
3. Select Visit Label and click on the 👉 Upload file `again` - A popup should prompt you to select a Site as it is a required field ❌

Comment (maltheism) [reply modules/media/test/TestPlan.md:56]: @christinerogers would you happen to have an opinion on this?

**Comment by johnsaigle** [comment]
Closing and re-opening to restart Travis. It looks like it has a bug for this build.

File: modules/media/test/TestPlan.md (hunk 1 above)
Comment (christinerogers) [reply modules/media/test/TestPlan.md:56]: Whether we should remove the word `again/one more time` or not?  Sure, remove it.

**Comment by kongtiaowang** [comment]
I have restart your Travis, it should work for you.

**Comment by driusan** [comment]
@maltheism What's the status of this PR? It looks like you reviewed it and the changes were done but it's not approved

**Comment by maltheism** [comment]
@driusan everything looks good now. Sorry I somehow missed the notification from you until now.

**Pull request merged**